Gopkg.toml

```diff
@@ -1,19 +1,7 @@
-[[constraint]]
```

Comment:
Why were these constraints removed? I just rebuilt the file (using `dep init`) and with a few changes got this:

```toml
[prune]
  unused-packages = true
  non-go = true
  go-tests = true

  [[prune.project]]
    name = "github.com/grpc-ecosystem/grpc-gateway"
    unused-packages = false
    non-go = false

[[constraint]]
  name = "github.com/gogo/protobuf"
  version = "1.0.0"

[[constraint]]
  name = "github.com/golang/protobuf"
  version = "1.0.0"

[[constraint]]
  name = "github.com/grpc-ecosystem/grpc-gateway"
  version = "1.3.1"

[[constraint]]
  name = "github.com/magefile/mage"
  version = "2.0.1"

[[constraint]]
  name = "github.com/pkg/errors"
  version = "0.8.0"

[[constraint]]
  name = "github.com/segmentio/ksuid"
  version = "1.0.1"

[[constraint]]
  name = "google.golang.org/grpc"
  version = "1.9.2"

[[constraint]]
  name = "zvelo.io/zmage"
  version = "3.0.1"
```